.github/workflows/cli-scripts-deploy-managed-online-endpoint.yml

```diff
@@ -10,6 +10,8 @@ on:
       - cli/deploy-managed-online-endpoint.sh
       - .github/workflows/cli-scripts-deploy-managed-online-endpoint.yml
       - cli/setup.sh
+      - cli/endpoints/online/managed
```

Comment:
Needs to be this: endpoints/online/managed/sample
The above path lots of stuff not needed by this script